packages/client/src/components/ClientProvider/ClientProvider.tsx

```diff
@@ -69,7 +69,7 @@ const ClientProvider = ({
       client ??
       createClient({
         url: `${apiHost}${apiUri}`,
-        requestPolicy: "cache-and-network",
+        requestPolicy: "cache-first",
```

Comment:
Without this change, the cache is never read. Not sure if we really want this :woman_shrugging: